Update quickstart to add reference to `--default-repo` (#4781)
GoogleContainerTools/skaffold: docs/content/en/docs/quickstart/_index.md

```

{{< alert title="Note">}}

> If you are deploying to a remote cluster, you must run `skaffold dev --default-repo=gcr.io/k8s-skaffold/getting-started`
> where the repository is a location that you have write-access to.  Skaffold then

Reviewer: ```suggestion
If you are deploying to a remote cluster, you must run `skaffold dev --default-repo=<my_registry>`
where `<my_registry>` is an image registry that you have write-access to. Skaffold then
```
